common/src/main/java/org/candlepin/common/jackson/DynamicPropertyFilter.java

```diff
@@ -28,12 +32,27 @@
  */
 public class DynamicPropertyFilter extends CheckableBeanPropertyFilter {
 
+    private List<String> path = new ArrayList<String>(10);
+
     public boolean isSerializable(Object obj, JsonGenerator jsonGenerator,
         SerializerProvider serializerProvider, PropertyWriter writer) {
-        DynamicFilterData filterData =
-            ResteasyProviderFactory.getContextData(DynamicFilterData.class);
+
+        DynamicFilterData filterData = ResteasyProviderFactory.getContextData(DynamicFilterData.class);
+
         if (filterData != null) {
-            return !filterData.isAttributeExcluded(writer.getName(), obj);
+            this.path.clear();
```

Comment:
We can define the 'path' collection in this method so that we didn't have to clear it each time. Any reason that you define path collection as a field?